bookwyrm/static/js/forms.js

```diff
@@ -46,4 +46,10 @@
     document
         .querySelectorAll("[data-remove]")
         .forEach((node) => node.addEventListener("click", removeInput));
+
+    document.getElementById("id_description").addEventListener("keypress", function (event) {
```

Comment:
Your javascript looks perfect, but I think it should be applied to `id_add_subject` and the created fields (`id_add_subject-2`, etc) rather than the description field. As it is, this prevents adding line breads to book descriptions.

Reply:
I updated the js - it now uses `querySelectorAll("[name='subjects']")`. 

All other fields still save by hitting enter (except description)